### ISSUE TITLE
SSH client wont open

### ISSUE THREAD

**Issue by elch0101** [microsoft/vscode-remote-release#2959]
Issue Type: <b>Bug</b>

Im trying to open a ssh client but every time i write the password the console clears and asks me again for my password, it doesnt tell me my password is wrong, it just clears and i cant connect

VS Code version: Code 1.45.0 (d69a79b73808559a91206d73d7717ff5f798f23c, 2020-05-07T16:18:48.860Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-9400F CPU @ 2.90GHz (6 x 2904)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.93GB (12.51GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (25)</summary>

Extension|Author (truncated)|Version
---|---|---
html-snippets|abu|0.2.1
vscode-intelephense-client|bme|1.3.11
c-cpp-compile-run|dan|1.0.8
vscode-html-css|ecm|0.2.3
prettier-vscode|esb|4.5.0
php-debug|fel|1.13.0
php-intellisense|fel|2.3.14
code-runner|for|0.10.0
vscode-clang|mit|0.2.3
csharp|ms-|1.21.18
remote-containers|ms-|0.117.0
remote-ssh|ms-|0.51.0
remote-ssh-edit|ms-|0.51.0
remote-wsl|ms-|0.44.2
vscode-remote-extensionpack|ms-|0.20.0
cpptools|ms-|0.27.1
material-icon-theme|PKi|4.1.0
java|red|0.61.0
phpdoc-comment-vscode-plugin|rex|1.3.0
vscodeintellicode|Vis|1.2.7
vscode-java-debug|vsc|0.25.1
vscode-java-dependency|vsc|0.9.0
vscode-java-pack|vsc|0.9.0
vscode-java-test|vsc|0.22.3
vscode-maven|vsc|0.21.3

(1 theme extensions excluded)

</details>

**Comment by elch0101** [comment]
> Are you sure that you are logging in as the right user? Do you have an ssh config set up? If it's not using the "User" set in an ssh config it will use the user name of your local machine which sometimes leads to confusion.

Yes, im logging in as the right user, i have an ssh config setup. I was using it yesterday and it was working, but today i cant log in, and i loged in via ssh in secure shell just to make sure it was not the server's fault and i was able to log in without problems.

**Comment by elch0101** [comment]
> Are you sure that you are logging in as the right user? Do you have an ssh config set up? If it's not using the "User" set in an ssh config it will use the user name of your local machine which sometimes leads to confusion.


I was able to record what the terminal says before it "restarts", do you have any email you can give me to send you the video? (it has personal data)

**Comment by Alvindelito** [comment]
I'm getting the same problem. Was SSHed into my server with VScode yesterday and can't connect at all this morning. I can connect perfectly fine from my terminal with the same login credentials.

![vscode error](https://user-images.githubusercontent.com/17840013/81427757-140b2380-9110-11ea-8f02-114697a880bd.jpg)

**Comment by tspriggs** [comment]
I have recently setup an SSH connection to my university server, which was working perfectly 2 days ago. Then there was an update to VScode which I applied before heading offline. The next day, I could not connect to the SSH client as I was doing the previous day.
When attempting to login to the SSH server, I was getting a notification in the lower right hand corner, saying it as attempting to connect, though it was retrying, followed by the attempt number.
The output was wiped before I can copy it to post.
When I downgraded VScode to 1.44 from last month, the problem disappears and I can SSH as normal. However when it updated again, the SSH connection wouldn't complete again.
